From 416243eee9b03d2dd90cbc0174836d4cd1f97af2 Mon Sep 17 00:00:00 2001
From: Priyanka Bose <priyanka.bose@intel.com>
Date: Wed, 19 Sep 2018 23:11:23 +0530
Subject: [PATCH] Adding dependancies for Capstone Disassembler in Vendor Art

Change-Id: I280791e6904ff8b94d577c66c084fe1661652b45
Category: device enablement
Domain: AOSP.ART-Other
Origin: internal
Upstream-Candidate: no, proprietary
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-68865
Signed-off-by: Priyanka Bose <priyanka.bose@intel.com>
---
 android/config.go   |  4 ++++
 android/variable.go | 10 ++++++++++
 2 files changed, 14 insertions(+)

diff --git a/android/config.go b/android/config.go
index 06169de9..50ebe8af 100644
--- a/android/config.go
+++ b/android/config.go
@@ -849,3 +849,7 @@ func stringSlice(s *[]string) []string {
 		return nil
 	}
 }
+
+func (c *config) AutoFastJni() bool {
+        return Bool(c.productVariables.AutoFastJni)
+}
diff --git a/android/variable.go b/android/variable.go
index 2eed3458..4aa5fb7c 100644
--- a/android/variable.go
+++ b/android/variable.go
@@ -100,6 +100,14 @@ type variableProperties struct {
 			Enabled *bool `android:"arch_variant"`
 		} `android:"arch_variant"`
 
+
+		AutoFastJni struct {
+                        Srcs          []string
+                        Include_dirs  []string
+                        Static_libs   []string
+                        Shared_libs   []string
+                }
+
 		Uml struct {
 			Cppflags []string
 		}
@@ -237,6 +245,8 @@ type productVariables struct {
 
 	VendorVars map[string]map[string]string `json:",omitempty"`
 
+	AutoFastJni *bool `json:",omitempty"`
+
 	// include Bliss variables
 	Bliss android.ProductVariables
 }
-- 
2.17.1

